src/packages/core/localization/manifests.ts

```diff
@@ -0,0 +1,27 @@
+import { ManifestTypes } from '../extension-registry/index.js';
+import { ManifestTranslations } from '../extension-registry/models/translations.model.js';
+
+const translationManifests: Array<ManifestTranslations> = [
+	{
+		type: 'translations',
+		alias: 'Umb.Translations.En_US',
+		weight: -100,
```

Comment:
I'm not sure what the weight will achieve. Shouldn't we just get rid of it? :-)

Reply:
The idea is that we are sure to load the system languages first since we want any extension languages to override ours. But I'm not sure if the weight is used automatically or not.

Reply:
interesting approach — as I was only seeing weight as a sorting.

So I'm not sure the weight would have an influence on that, but I will check. :-)